turtlebot4_bringup/launch/robot.launch.py

```diff
@@ -63,9 +73,19 @@ def generate_launch_description():
         output='screen',
         condition=LaunchConfigurationEquals('model', 'standard')
     )
+    create3_republisher_node = Node(
+        package='create3_republisher',
+        executable='create3_republisher',
+        parameters=[create3_repub_param_yaml_file,
+                    {'robot_namespace': '_do_not_use'}],
```

Comment:
I had to explicitly add the forward slash to the topic name here, i.e.:

`{'robot_namespace': '/_do_not_use'}],`

Without it, the create3 republisher node failed to start.